Update DESTINATION_TEMPLATE.mdx (#904)
rudderlabs/rudderstack-docs: DESTINATION_TEMPLATE.mdx

---
title: "Destination_Name"
description: Step-by-step guide on sending your event data from RudderStack to **Destination_Name**.

> aliases: ["dest_name","dest_name_2"]

Reviewer: Can you add a comment here about what aliases are? Something like

"Alias values are used to optimize the page in our search index."

Reviewer 2: makes sense, added this bit.

> aliases: ["dest_name","dest_name_2"]

Reviewer 2: ```suggestion
aliases: ["dest_name","dest_name_2"]  // These alias values are used to optimize the page our search index, making it easier for users to find the integration in the docs.
```

> - Do not include any screenshots/images. We will include them after merging the changes.
> - Include tips and suggestions wherever possible.

[Destination](Link)- Add some introduction about the destination platform.

<div class="infoBlock">
Find the open source transformer code for this destination in the <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/<dest_name>">GitHub repository</a>.
</div>

## Getting started

> RudderStack supports sending events to <Destination_Name> via the following <a href="https://rudderstack.com/docs/rudderstack-cloud/rudderstack-connection-modes/">connection modes</a>:

Reviewer 2: ```suggestion
RudderStack supports sending events to <Destination_Name> via the following <Link to="/destinations/rudderstack-connection-modes/">connection modes</Link>:
```

> RudderStack supports sending events to <Destination_Name> via the following <a href="https://rudderstack.com/docs/rudderstack-cloud/rudderstack-connection-modes/">connection modes</a>:

Reviewer 2: agreed - added!

| **Connection Mode** | **Web**       | **Mobile**    | **Server**    |
| :------------------ | :------------ | :------------ | :------------ |
| **Device mode**     |               |               |               |
| **Cloud mode**      |               |               |               |

Once you have confirmed that the source platform supports sending events to <Destination_Name>, follow these steps:

1. From your [RudderStack dashboard](https://app.rudderstack.com/), add the source. Then, from the list of destinations, select **<Destination_Name>**.
2. Assign a name to your destination and click **Continue**.

### Connection settings 

> To successfully configure Mixpanel as a destination, you need to configure the following settings:

Reviewer 2: ```suggestion
To successfully set up Mixpanel as a destination, you need to configure the following settings:
```

Reviewer 2: As "configure" is repeated.
